src/styles/clock.scss

```
@@ -0,0 +1,43 @@
+.clock {
+  position: relative;
+  @include center-block;
+
+  &__base {
+    position: absolute;
+    height: 50px;
+    width: 50px;
+    border-radius: 50%;
+    background-color: white;
+    z-index: 2;
+  }
+
+  &__minutes {
+    position: absolute;
+    bottom: 50%;
+    height: 400px;
+    width: 20px;
+    border-radius: 3px;
+    background-color: white;

```

create vars for colors
```suggestion
    background-color: $c-white;

```